frontend/src/pages/destructionlist/detail/components/DestructionListProcessZaakReviewModal/DestructionListProcessZaakReviewModal.tsx

```diff
@@ -192,10 +200,18 @@ export const DestructionListProcessZaakReviewModal: React.FC<
       },
 
       {
-        label: isArchiefactiedatumActive ? "archiefactiedatum" : undefined,
+        label:
+          isArchiefactiedatumActive &&
+          !getBewaartermijn(_formState.selectielijstklasse)
```

Comment:
This should be the opposite. If a selectielijstklasse has "waardering" `blijvend_bewaren`, then it will have no archiefactiedatum (no bewaartermijn). If there is no bewaartermijn, the archiefactiedatum field should **not** be visible.

```suggestion
          getBewaartermijn(_formState.selectielijstklasse)
```